### DISCUSSION TITLE
Google Page Speed Insights

### DISCUSSION THREAD

**Discussion by DigitalTrading** [dnnsoftware/Dnn.Platform#5304] [Q&A]
Hi,
I am looking at ways to create a new base DNN site which has a reasonable page speed score out of the box based on the mobile results. I have managed to get the desktop to have a performance of 94/100 but the mobile I am struggling to get anything over about 60. The issues typically relate to render blocking resources, FCP and LCP.
Has anyone got any suggestions on how to resolve these issues or are there any plans to help improve the core DNN build.

Thanks 
Roger

**Comment by mitchelsellers** [comment]
There is a lot to go with this. However, I would encourage posting this on the DNNCommunity.org site as GitHub is more for implementation details within the platform.  YOu will have better visibility to folks that can answer this there.

**Reply by DigitalTrading** [reply]
Thanks I have moved to the community site.
